traitsui/tests/editors/test_instance_editor.py

```diff
@@ -19,28 +20,59 @@ class EditedInstance(HasTraits):
 
 class NonmodalInstanceEditor(HasTraits):
     inst = Instance(EditedInstance, ())
-    traits_view = View(Item("inst", style="simple"), buttons=["OK"])
 
 
+def get_view(style):
+    return View(Item("inst", style=style), buttons=["OK"])
+
+
+@requires_toolkit([ToolkitName.qt, ToolkitName.wx])
 class TestInstanceEditor(unittest.TestCase):
 
-    @requires_toolkit([ToolkitName.qt])
-    def test_simple_editor(self):
+    def check_editor_update_value(self, style):
         obj = NonmodalInstanceEditor()
-        with reraise_exceptions(), create_ui(obj) as ui:
-            editor = ui.get_editors("inst")[0]
+        tester = UITester()
+        with tester.create_ui(obj, dict(view=get_view(style))) as ui:
+            instance = tester.find_by_name(ui, "inst")
+            editor = instance.locate(locator.NestedUI())
+            value_txt = editor.find_by_name("value")
+            value_txt.perform(command.KeySequence("abc"))
+            if style == 'simple':
+                # the below code requires ui_base implementation to work
+                # ok_button = editor.find_by_id("OK")
+                # ok_button.perform(command.MouseClick())
+                press_ok_button(editor.target)
```

Comment:
This action looks redundant: the trait should be updated even without pressing the OK button, unless we need to switch focus for wx to trigger the change?